app/controllers/tweet.js

```diff
@@ -40,19 +40,44 @@ exports.getSearch = function(req, res) {
     });
 }
 
+// Request saved tweets from stream.
+// Return results from database
 exports.getStream = function(req, res) {
-    console.log('Twitter stream requested')
+    console.log('Fetching Cached Twitter Streams\n')
+
+    // Parameters to restrict the stream results to.
+    var params = {
+        track: '#' + req.params.hash,
+    };
+
+    Tweet.find(function(err, tweets) {
+        if (err) res.send(err);
+        res = setHeaders(res);
+        res.json(tweets);
+        console.log('Tweets sent.');
+    });
+}
+
+// Create a new twitter stream.
+// This saves tweets under the provided
+// hashtag to our database
+exports.createStream = function(req, res) {
+    console.log('Initialize Twitter Stream. Recording incoming tweets.\n')
+    
     // Parameters to restrict the stream results to.
     var params = {
         track: '#' + req.params.hash,
         // 4 mile radius around downtown Boston.
         geocode: "42.351252, -71.073808, 4mi",
     };
+    
+    res = setHeaders(res);
+    res.json('{Stream created, recording tweets for: ' + req.params.hash + '}');
```

Comment:
res.json({ message : "Stream created, recording tweets for: " + req.params.hash });